src/app/component/RecentlyViewedWidget/RecentlyViewedWidget.container.js

```diff
@@ -22,4 +23,60 @@ export const mapStateToProps = (state) => ({
 // eslint-disable-next-line no-unused-vars
 export const mapDispatchToProps = (dispatch) => ({});
 
-export default connect(mapStateToProps, mapDispatchToProps)(RecentlyViewedWidget);
+/** @namespace Component/RecentlyViewedWidget/Container */
+export class RecentlyViewedWidgetContainer extends PureComponent {
+    state = {
+        siblingsHaveBrands: false,
+        siblingsHavePriceBadge: false,
+        siblingsHaveTierPrice: false,
+        siblingsHaveConfigurableOptions: false
+    };
+
+    productCardFunctions = {
+        setSiblingsHaveBrands: this.setSiblingsHaveBrands.bind(this),
+        setSiblingsHavePriceBadge: this.setSiblingsHavePriceBadge.bind(this),
+        setSiblingsHaveTierPrice: this.setSiblingsHaveTierPrice.bind(this),
+        setSiblingsHaveConfigurableOptions: this.setSiblingsHaveConfigurableOptions.bind(this)
+    };
+
+    setSiblingsHaveBrands() {
+        this.setState({ siblingsHaveBrands: true });
+    }
+
+    setSiblingsHavePriceBadge() {
+        this.setState({ siblingsHavePriceBadge: true });
+    }
+
+    setSiblingsHaveTierPrice() {
+        this.setState({ siblingsHaveTierPrice: true });
+    }
+
+    setSiblingsHaveConfigurableOptions() {
+        this.setState({ siblingsHaveConfigurableOptions: true });
+    }
+
+    render() {
+        const {
+            siblingsHaveBrands,
+            siblingsHavePriceBadge,
+            siblingsHaveTierPrice,
+            siblingsHaveConfigurableOptions
+        } = this.state;
+
+        const productCardProps = {
+            siblingsHaveBrands,
+            siblingsHavePriceBadge,
+            siblingsHaveTierPrice,
+            siblingsHaveConfigurableOptions
+        };
```

Comment:
Same here as above

Reply:
Moved product cart props to the containerProps().